Portfolio/js/PortfolioVersion2.js

```diff
@@ -0,0 +1,91 @@
+(function () {
+    var $portfolio = document.getElementById("portfolio");
+    var $btnLeft = $portfolio.querySelector(".arrow-left");
+    var $btnRight = $portfolio.querySelector(".arrow-right");
+    var $carousel = $portfolio.querySelector(".nav-tab-wrap");
+    var $galleryItems = $portfolio.querySelectorAll(".item");
+    var $tabs = document.querySelectorAll(".tab");
+    var VISIBLE_ITEMS = 3;
+    var CLASS_VISIBLE = "visible";
+    var CLASS_ACTIVE = "active";
+    var baseItemWidth = $galleryItems[0].getBoundingClientRect().width;
+    var itemWidth = 0;
+
+    $btnRight.addEventListener("click", function () {
```

Comment:
it's better to move it into a separte function. something like `addEventsListeners() {...}`